lib/fwupd_l10n.dart

```diff
@@ -142,39 +142,39 @@ extension FwupdExceptionL10n on FwupdException {
   String localize(BuildContext context) {
     final l10n = AppLocalizations.of(context);
     switch (runtimeType) {
-      case FwupdInternalException:
+      case const (FwupdInternalException):
```

Comment:
Nitpick: it might be worth using Dart's new switch expression here to make it more compact/readable.